github/ci/prow-deploy/molecule/default/prepare.yml

```diff
@@ -1,15 +1,87 @@
-- hosts: localhost
+- hosts: instance
   tasks:
     - set_fact:
-        deploy_environment: '{{ lookup("env", "DEPLOY_ENVIRONMENT") }}'
-    - fail:
+        deploy_environment: '{{ lookup("env", "DEPLOY_ENVIRONMENT") | default("kubevirtci-testing")}}'
+    - name: Check DEPLOY_ENVIRONMENT env var
```

Comment:
If you are setting a default then this check will never fail

Reply:
Makes sense, removed